doc/mb_compare.md

```diff
@@ -0,0 +1,34 @@
+# MirrorCache (mc) concept vs MirrorBrain (mb) concept
+
+1. Scanning
+1.1
+mb uses daily full scan of all files on all mirrors, without regard of whether files were ever downloaded or changed recently. This may be waste of resources.
```

Comment:
we have 2 modes:

1. scan only the directories where we know a change occured. (repo-pusher)
2. scan regularly to detect changes done by the mirror admin. scan mirrors were we do not push (yet)

Reply:
The statement above doesn't contradict this.